setup.py

```diff
@@ -3,12 +3,12 @@
 
 def requirements() -> list:
     return [
-        'click==6.7',
-        'curio==0.8',
-        'asks==1.3.6',
-        'yarl==0.15.0',
-        'beautifulsoup4==4.6.0',
-        'tabulate==0.8.2',
+        'click',
```

Comment:
mmm, I don't like not freezing deps - builds can suddenly start failing for the same commit, hence worse reproducibility :/